File: Selenium/Project/Activity1.java
package groupProject;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args){
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        driver.get("https://alchemy.hguy.co/jobs");
        String title = driver.getTitle();
        System.out.printf(title);
        if(title.equalsIgnoreCase(title)){
            System.out.printf("title matched");
        }
        else{
            System.out.printf("title not matched");
        }

        driver.close();
    }
}

File: Selenium/Project/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver.firefox import GeckoDriverManager
​
service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://alchemy.hguy.co/jobs/")
    title = driver.getTitle();
    print(title)
    if title == "Alchemy Jobs – Job Board Application":
        print("title matched")
        driver.close()
    else:
        print("title  not matched")